package/run.sh

```diff
@@ -131,12 +204,21 @@ standalone_node() {
     rolling_backup &
     etcd \
         --name ${NAME} \
-        --listen-client-urls http://0.0.0.0:2379 \
-        --advertise-client-urls http://${IP}:2379 \
-        --listen-peer-urls http://0.0.0.0:2380 \
-        --initial-advertise-peer-urls http://${IP}:2380 \
-        --initial-cluster ${NAME}=http://${IP}:2380 \
-        --initial-cluster-state new
+        --listen-client-urls https://0.0.0.0:2379 \
+        --advertise-client-urls https://${ETCD_CONTAINER_NAME}:2379 \
+        --listen-peer-urls https://0.0.0.0:2380 \
+        --initial-advertise-peer-urls https://${ETCD_CONTAINER_NAME}:2380 \
+        --initial-cluster ${NAME}=https://${ETCD_CONTAINER_NAME}:2380 \
+        --initial-cluster-state new \
+        --client-cert-auth \
+        --peer-client-cert-auth \
+        --trusted-ca-file $ETCD_CA_FILE \
+        --key-file $ETCD_KEY_FILE \
+        --cert-file $ETCD_CERT_FILE \
```

Comment:
where exactly do you initialize this environment variable

Reply:
Passed through the template

Reply:
Moved to the script.